TACC/Core-CMS-Custom
# task/WP-329 and bug/WP-330: Fixes for registration form

## Overview
Re-numbers superscript references on form + fixes validation on phone and email fields for contacts
…

## Related

- [WP-329](https://jira.tacc.utexas.edu/browse/WP-329)
- [WP-330](https://jira.tacc.utexas.edu/browse/WP-330)
…

## Changes
- Removed unused superscripts 2, 3, and 4 from bottom of form and re-numbered 5, 6, and 7 to 2, 3, and 4
- In regex used to validate phone numbers and email addresses for contacts, escaped `-` characters within character sets to prevent invalid regex from checking these fields at all
…

## Testing

1. Go to https://localhost:8000/register/request-to-submit/
2. Scroll to the bottom of the form and confirm there are only 4 entries listed for superscript descriptions
3. Confirm, in the form, that these descriptions correctly match where said superscripts are used (the first is used in the 'Type' drop down in the 'Organization' section of the form)
4. Fill out the form, and once at Contacts, try putting malformed phone numbers and emails in at least two contact entries
5. Confirm you cannot submit the form unless these fields are formatted correctly
6. After submission, go to https://localhost:8000/administration/list-registration-requests/ and confirm your new record is on the listing and, via View Record, all data submitted shows up
7. Now, on the same record, via Edit Record, re-test steps 4-6

## UI

…
